config/initializers/filter_parameter_logging.rb

```diff
@@ -4,5 +4,9 @@
 # Use this to limit dissemination of sensitive information.
 # See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += [
-  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+  :passw, :email, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
 ]
+
+# WARNING: Due to https://github.com/rails/rails/issues/48704, the filter parameters defined here do not get applied to
```

Comment:
Let's avoid setting this list in 2 places. Should we remove `config.filter_parameters` from application.rb, and keep the list here with the addition from https://github.com/rails/rails/issues/48704#issuecomment-1873236944?